core/src/main/java/org/apache/iceberg/MetadataUpdate.java

```diff
@@ -216,19 +216,19 @@ public void applyTo(TableMetadata.Builder metadataBuilder) {
   }
 
   class RemoveSnapshotRef implements MetadataUpdate {
-    private final String name;
+    private final String refName;
```

Comment:
👍  This change is good. If somebody were to implement their own small enums in another language or with something that could go from kebab-case to camel-case automatically, then this change would be sufficient to allow them to update the parser themselves.